packages/hop-node/src/watchers/CommitTransferWatcher.ts

```diff
@@ -219,21 +221,21 @@ class CommitTransfersWatcher extends BaseWatcherWithEventHandlers {
         `sending commitTransfers (destination chain ${destinationChainId}) tx`
       )
 
-      this.isCommitTxSent[destinationChainId] = true
+      this.commitTxSentAt[destinationChainId] = Date.now()
       const tx = await l2Bridge.commitTransfers(destinationChainId)
       tx?.wait()
         .then(async (receipt: providers.TransactionReceipt) => {
           if (receipt.status !== 1) {
-            this.isCommitTxSent[destinationChainId] = false
+            this.commitTxSentAt[destinationChainId] = 0
             throw new Error('status=0')
           }
           this.emit('commitTransfers', {
             destinationChainId
           })
-          this.isCommitTxSent[destinationChainId] = false
+          this.commitTxSentAt[destinationChainId] = 0
```

Comment:
If the tx is successful then there's no need to reset the timestamp. Just wondering if the tx will be re-attempted if it's reset here before the watcher receives the event

Reply:
If we do not reset it, we will always have to wait `TX_RETRY_DELAY_MS` (not the worst thing). I believe resetting it will allow us to optimally commit transfers (i.e. commit transfers twice in 2 minutes if the min commit transfers is hit during that small timeframe).

> Just wondering if the tx will be re-attempted if it's reset here before the watcher receives the event
I don't think it will because the onchain pending amt will be `0` and the poller will `return` [here[(https://github.com/hop-protocol/hop/blob/develop/packages/hop-node/src/watchers/CommitTransferWatcher.ts#L193-L205).

Reply:
Sounds good 👍